exec/java-exec/src/main/resources/ace.mode-sql.template.js

```diff
@@ -0,0 +1,121 @@
+/**
```

Comment:
1. I suggest we keep js scripts / templates under one folder.
Could you please move it rest/static/js folder?
Also could you please do the same for options.dscribe.template.js

2. mode-sql.js has the same code, please factor out the same parts.